index.html

```diff
@@ -11,8 +11,8 @@
       <div class="header-container-inner">
         <div class="logo-container"></div>
```

Comment:
```suggestion
         <a class="logo-container" href="/"></a>
```
로고를 누르면 main 으로 가게 하면 좋을 것 같네요.

Reply:
수정하였습니다.